frontend/public/components/utils/details-page.tsx

```diff
@@ -78,6 +79,16 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({
           />
         </DetailsItem>
       )}
+      {customResource && (
```

Comment:
I wouldn't add a `customResource` to the generic summary component. Just make the additional details items children of the component like we do in other places.

If we *really* don't like the ordering it gives, I would just not use the `ResourceSummary` on the CSV page.